en/sensor/optical_flow.md

```diff
@@ -0,0 +1,51 @@
+# Optical Flow
+
+Optical Flow uses a downward facing camera and a downward facing distance sensor for position estimation. 
+Optical Flow based navigation is supported by: EKF2 and LPE.
+
+## Setup
+
+An Optical Flow setup requires a downward facing camera which publishes to the [OPTICAL_FLOW_RAD](https://mavlink.io/en/messages/common.html#OPTICAL_FLOW_RAD) topic and a [distance sensor](../sensor/rangefinders.md) (preferably a LiDAR) publishing messages to the [DISANCE_SENSOR](https://mavlink.io/en/messages/common.html#DISTANCE_SENSOR) topic.
+
+The output of the flow has to be as follows
+
+| Vehicle movement | Integrated flow |
+| -- | -- |
+| Forwards | + Y |
+| Backwards | - Y |
+| Right | - X |
+| Left | + X |
+
+For pure rotations the `integraded_xgyro` and `integraded_x` (respectively `integraded_ygyro` and `integraded_y`) have to be the same.
```

Comment:
What are integraded_xgyro and integraded_x? 
Also, are these miss-spelled or do they reflect code?

Reply:
Yes, but I mean are these miss-spelled here? Ie it is `integraDed_xgyro` above, but integraTed_xgyro` in [OPTICAL_FLOW_RAD](https://mavlink.io/en/messages/common.html#OPTICAL_FLOW_RAD)?

Reply:
Ah, right. Yes they are simply miss-spelled.